DSharpPlus.CommandsNext/EventArgs/CommandContext.cs

```diff
@@ -97,61 +97,25 @@ internal CommandContext()
         /// Quickly respond to the message that triggered the command.
         /// </summary>
         /// <param name="content">Message to respond with.</param>
-        /// <param name="isTTS">Whether the message is to be spoken aloud.</param>
-        /// <param name="embed">Embed to attach.</param>
-        /// <param name="mentions">A list of mentions permitted to trigger a ping.</param>
         /// <returns></returns>
-        public Task<DiscordMessage> RespondAsync(string content = null, bool isTTS = false, DiscordEmbed embed = null, IEnumerable<IMention> mentions = null) 
-            => this.Message.RespondAsync(content, isTTS, embed, mentions);
-
-        /// <summary>
-        /// Quickly respond with a file to the message that triggered the command.
-        /// </summary>
-        /// <param name="fileName">Name of the file to send.</param>
-        /// <param name="fileData">Stream containing the data to attach as a file.</param>
-        /// <param name="content">Message to respond with.</param>
-        /// <param name="isTTS">Whether the message is to be spoken aloud.</param>
-        /// <param name="embed">Embed to attach to the message.</param>
-        /// <param name="mentions">A list of mentions permitted to trigger a ping.</param>
-        /// <returns>Message that was sent.</returns>
-        public Task<DiscordMessage> RespondWithFileAsync(string fileName, Stream fileData, string content = null, bool isTTS = false, DiscordEmbed embed = null, IEnumerable<IMention> mentions = null) 
-            => this.Message.RespondWithFileAsync(fileName, fileData, content, isTTS, embed, mentions);
```

Comment:
Ehhh, I'm not sure removing these is a good idea right now considering how often they are used. I would keep the message builder overload, but perhaps use an obsolete attribute on this to recommend the message builder instead.

Reply:
@Emzi0767 thoughts on this?